.github/workflows/ci.yml

```diff
@@ -7,17 +7,26 @@ on:
     branches:
       - '*'
 jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: 🧼 lint renovate config # Validates changes to renovate.json config file
+        uses: suzuki-shunsuke/github-action-renovate-config-validator@v0.1.2
+        with:
+          config_file_path: 'renovate.json'          
   ci-build:
     name: Run Build
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: set up JDK 11
+        uses: actions/setup-java@v3
         with:
-          java-version: 1.8
+          distribution: 'corretto'
```

Comment:
curious why you chose the Corretto distribution?

Reply:
No specific reason. changing to `temurin`

Reply:
And next question is why Temurin? 😹

Reply:
I have updated plugin `actions/setup-java@v3` Here we need to specify JDK [distribution](https://github.com/actions/setup-java#supported-distributions) name. temurin is new  name for `AdoptOpenJDK` which is used in wayfair.

Reply:
ahh..that makes sense.  Should probably add a comment to the YAML file so folks know what it means when looking at this later IMHO.